webportal/lambda/deployment/gateway_normalizer.py

```diff
@@ -0,0 +1,50 @@
+import boto3
+from . import chalice_config_reader
+
+LOG_FORMAT = "$context.identity.sourceIp $context.identity.caller $context.identity.user [$context.requestTime] \"$context.httpMethod $context.resourcePath $context.protocol\" $context.status $context.responseLength $context.requestId"
+
+
+def normalize_gateway(environment):
+    rest_api_config = chalice_config_reader.find_deployed_config('rest_api', environment)
+    rest_api_id = rest_api_config['rest_api_id']
+    chalice_config = chalice_config_reader.chalice_config()
+    stage_name = chalice_config['stages'][environment]['api_gateway_stage']
+
+    gateway_client = api_gateway_client()
+    webportal_name = f'{environment}-webportal'
+    gateway_client.update_rest_api(restApiId=rest_api_id,
+                                   patchOperations=api_gateway_patch_operations(webportal_name))
+    gateway_client.update_stage(restApiId=rest_api_id, stageName=stage_name,
+                                patchOperations=stage_patch_operations(rest_api_id, stage_name))
+    gateway_client.tag_resource(resourceArn=f'arn:aws:apigateway:{get_region()}::/restapis/{rest_api_id}',
+                                tags={"Environment": environment, "Project": "SDC-Platform", "Team": "sdc-platform"})
```

Comment:
The tags are copy/pasted all over. Don't have a solution, but I suspect if they need to change in the future, we will miss something.

Reply:
You are correct, they are copy pasted all over... Terraform, json files, and now here. The only solace I can give you is that they are documented here:
https://securedatacommons.atlassian.net/wiki/spaces/SD/pages/247562350/SDC+AWS+Resource+Tagging+Guide

Reply:
Didn't know that wiki existed

Reply:
Written by our own Tianna Woodson...